code/modules/mob/living/carbon/human/life.dm

```diff
@@ -609,7 +609,26 @@
 		return
 	//Stuff like the xenomorph's plasma regen happens here.
 	species.handle_environment_special(src)
-
+	if(isSynthetic()) // synth specific temperature values in the absence of a synthetic species
+		species.heat_level_1 = 400
+		species.heat_level_2 = 420 // haha nice
+		species.heat_level_3 = 1000
+		species.cold_level_1 = 275
```

Comment:
are synths not supposed to be totally immune to cold? i swear this was a thing

Reply:
> are synths not supposed to be totally immune to cold? i swear this was a thing

i dont mind changing that if u can get a maintainer's go ahead for it